src/js/redux/reducers/bulkDownload/bulkDownloadReducer.js

```diff
@@ -25,6 +25,13 @@ export const initialState = {
             id: '',
             name: 'Select a Sub-Agency'
         },
+        location: {
+            country: {
+                code: '',
+                name: ''
+            },
+            state: ''
```

Comment:
This should probably also be an object, in order to retain both the full name and state code.